### PR TITLE
Should use label 'Release' to set release

### PR DESCRIPTION
**Pull request by twaugh**

### CODE REVIEW

**Comment by vrutkovs** [comment]
LGTM, tested on local koji instance

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -605,7 +605,7 @@ def _get_labels_override(self, opts):
         labels = {}
         release = opts.get('release', None)
         if release:
-            labels['release'] = release
+            labels['Release'] = release
```
Comment (lcarva) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Remove lowercase `'release'` from `LABEL_NAME_MAP`? https://github.com/twaugh/koji-containerbuild/blob/6189ec047686f45f8dd43c807e5b07e705158af0/koji_containerbuild/plugins/builder_containerbuild.py#L75

Comment (twaugh) [inline koji_containerbuild/plugins/builder_containerbuild.py]: No, I think the rule until we switch to the new label set completely is:
* prefer the new set when reading labels
* use the old set when writing labels

**Comment by lcarva** [comment]
LGTM

**Pull request merged**